script/changelog

```diff
@@ -20,18 +20,50 @@ commit_summary() {
   echo "* $title #$id (@$author)"
 }
 
+revisions () {
```

Comment:
I'm wondering if there is another name we could use for `revisions()`. The reason I suggest this is that I was myself confused at what this function does. Perhaps instead we could call it:

- `revisions_in` (i.e., so that it would read `revisions_in $range`), or
- `get_revisions_in` (so that it is more clear that this method "looks something up").

What do you think?

Reply:
`revisions_in` sounds good.